app/src/main/java/de/westnordost/streetcomplete/quests/AbstractQuestAnswerFragment.kt

```diff
@@ -222,6 +222,18 @@ abstract class AbstractQuestAnswerFragment<T> :
         }
     }
 
+    protected fun createConvertToStepsAnswer(answer: T): AnswerItem? {
+        val way = osmElement as? Way ?: return null
+
+        if (way.isArea()) return null
+
+        if (!way.tags.containsKey("highway") || way.tags["highway"] == "steps") return null
+
+        return AnswerItem(R.string.quest_generic_answer_is_actually_steps) {
+            applyAnswer(answer)
+        }
+    }
+
```

Comment:
No, I meant not like that, but like any other other answer. Per quest. The only thing that is shared is the same string.

There shoiuld be no special provision in AbstractQuestAnswerFragment.

Reply:
Done in [`5b090e1` (#3334)](https://github.com/streetcomplete/StreetComplete/pull/3334/commits/5b090e1a78335a153a58e7f969185e972953978b)